### PR TITLE
Make tests pass, and actually run them on CircleCI.

### PR DESCRIPTION
**Pull request by Elarnon**
Tests were not actually running on CircleCI because `manage.py test` seems to require to be run in the current directory. See e.g. https://circleci.com/gh/mangaki/mangaki/64?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link and find the line "Ran 0 tests in 0.000s" in the output.

Moreover tests were not passing at all; this also fixes that so that they are not running.

Related to #214

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ test:
   pre:
     - python mangaki/manage.py collectstatic --no-input
   override:
-    - coverage run ./mangaki/manage.py test
+    - cd mangaki && coverage run manage.py test
```
Comment (RaitoBezarius) [inline circle.yml]: We need to move `.coveragerc` and similar files to the `mangaki` directory now, else, it won't be taken in account.

**Comment by Elarnon** [comment]
#214 should require advance notice to every users of Mangaki. Let's move the `.coveragerc` for now.

File: circle.yml (hunk 1 above)
Comment (Elarnon) [inline circle.yml]: Done.

**Comment by RaitoBezarius** [comment]
J'attends les tests et j'approuve :).

**Comment by RaitoBezarius** [comment]
Sad :(.

**Pull request merged**